contracts/carbon/CarbonController.sol

```diff
@@ -122,6 +122,13 @@ contract CarbonController is
         return _tradingFeePPM;
     }
 
+    /**
+     * @inheritdoc ICarbonController
+     */
+    function tradingFeePPMOverrides(uint128 pairId) external view returns (uint32) {
+        return _tradingFeePPMOverrides[pairId];
```

Comment:
Is override the right name for this? It could be just something like "custom fees" or even just "fees" (and then a default fee for everything that isn't defined explicitly).

Reply:
Changed to `customTradingFeePPM`